tachyon/math/elliptic_curves/affine_point.h

```diff
@@ -3,10 +3,8 @@
 
 #include <utility>
 
-#include "tachyon/base/buffer/copyable.h"
 #include "tachyon/math/elliptic_curves/jacobian_point.h"
 #include "tachyon/math/elliptic_curves/point_conversions_forward.h"
-#include "tachyon/math/geometry/point2.h"
 
```

Comment:
Please remove `point3.h` header (in `tachyon/math/elliptic_curves/jacobian_point.h` and `tachyon/math/elliptic_curves/projective_point.h`) and `point4.h` header (in `tachyon/math/elliptic_curves/point_xyzz_point.h`) as well.

Reply:
Also, could you remove these deps from the BUILD file as well?